ansible/roles/jitsi/tasks/main.yml

```diff
@@ -0,0 +1,104 @@
+---
```

Comment:
Is there a reason all the tasks in this role use `become: true`? It shouldn't be necessary as we specify it in our ansible.cfg

Reply:
Neh i think not, removed here a3eda10